modules/oadp-checking-api-group-versions.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/advanced-topics.adoc
+
+
+:_content-type: PROCEDURE
+[id="oadp-checking-api-group-versions_{context}"]
+= Listing the Kubernetes API group versions in a cluster
+
+A source cluster might offer multiple versions of an API, where one of these versions is the preferred API version. For example, a source cluster with an API named `Example` might be available in the `example.com/v1` and `example.com/v1beta2` API groups.
+
+If you use Velero to back up and restore such a source cluster, Velero backs up only the version of that resource that uses the preferred version of its Kubernetes API.
+
+In our case, if `example.com/v1` were the preferred API, then Velero would only back up the version of a resource that uses `example.com/v1`. Moreover, the target cluster would need to have `example.com/v1` registered in its set of available API resources in order for Velero to restore the resource on the target cluster.
```

Comment:
Do not use "our." This isn't one of the situations where the ISG allows personal pronouns.

Also, please don't use "would" here.
> ! Would v
Use only to refer to an unfulfilled condition or hypothesis and to express the future in the past, for example, "The command was rejected because it would have resulted in the ClusterName attribute being blank." Do not use as an auxiliary verb, for example, "You would need administrator authority to create an app." Instead, use the present tense, for example "You need administrator authority to create an app."

Reply:
Fixed; present simple used instead.